records_mover/creds/base_creds.py

```diff
@@ -107,15 +111,15 @@ def default_gcs_creds(self) -> Optional['google.auth.credentials.Credentials']:
             else:
                 creds = self.gcs(self._default_gcp_creds_name)
                 self.__default_gcs_creds = creds
-        except (OSError, google.auth.exceptions.DefaultCredentialsError):
+        except (OSError, google.auth.exceptions.DefaultCredentialsError) as e:
             # Examples:
             #   OSError: Project was not passed and could not be determined from the environment.
             #   google.auth.exceptions.DefaultCredentialsError: Could not automatically determine
             #     credentials. Please set GOOGLE_APPLICATION_CREDENTIALS or explicitly create
             #     credentials and re-run the application. For more information, please see
             #     https://cloud.google.com/docs/authentication/getting-started
-            logger.debug("google.cloud.storage not configured",
-                         exc_info=True)
+            logger.warning(f"google.cloud.storage not configured: {e}")
+            logger.debug("Details:", exc_info=True)
```

Comment:
The debugging here was kind of annoying (see the notes on the `GCP_PROJECT` environment variable), so I improved the logging a bit to help the next person.